src/PactNet.Native/Verifier/NativePactBrokerOptions.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using PactNet.Verifier;
+
+namespace PactNet.Native.Verifier
+{
+    /// <summary>
+    /// Native pact broker options
+    /// </summary>
+    internal class NativePactBrokerOptions : IPactBrokerOptions
+    {
+        private readonly IDictionary<string, string> verifierArgs;
+
+        /// <summary>
```

Comment:
why not add the inherited doc tag on the implementation side? more maintainable

Reply:
I really dislike that annotation - I want to see the docs whilst I'm in the code so that I know I'm keeping to them properly. That tag makes it far too easy to make that mistake.

Reply:
I understand why you would want to keep it in your implementation but 1) if any change, you have to change the base symbol first and then the implementation, strenuous to maintain, 2) Your code is your documentation, in principle, you should hot have to rely on the summary doc to give you extra information.

In anyway, this is not the scope of this PR but we should address it when we review the coding and naming standard.